.github/workflows/ci.yaml

```diff
@@ -33,7 +34,50 @@ jobs:
       - run: yarn install --immutable
       - run: yarn test
         working-directory: tests
-  cypress:
+
+  cypress-component:
+    strategy:
+      fail-fast: false
+      matrix:
+        browser: [chrome, firefox, electron]
+        count: [1]
+    name: Cypress component tests in ${{ matrix.browser }} ${{ matrix.count }}
+    timeout-minutes: 15
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 22
+          cache: yarn
+      - run: yarn set version stable
+      - run: yarn install --immutable
+
+      - run: yarn run cy:run:component --browser ${{ matrix.browser }}
+        working-directory: tests
+        env:
+          ELECTRON_ENABLE_LOGGING: true # send console logs to stdout in electron browser
+          NODE_ENV: test
+
+  cypress-component-rpc:
+    timeout-minutes: 5
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 22
+          cache: yarn
+      - run: yarn set version stable
+      - run: yarn install --immutable
+
+      - run: yarn run cy:run:rpc --browser electron # rpc test are expensive, so we stick to electron
+        working-directory: tests
+        env:
+          ELECTRON_ENABLE_LOGGING: true # send console logs to stdout in electron browser
+          NODE_ENV: test
```

Comment:
for RPC I guess we need the secrets being passed here

Reply:
I don't think anything needs to be passed. You can check `prices-api.yaml` where we use the NPM secret for publishing the package: `yarn config set npmAuthToken "${{ secrets.NPM_TOKEN }}"`